doc/contribute/guidelines.rst

```diff
@@ -727,151 +903,40 @@ workflow here:
-
-[area]
-  The ``[area]`` prefix usually identifies the area of code
-  being changed. It can also identify the change's wider
-  context if multiple areas are affected.
-
-  Here are some examples:
-
-  * ``doc: ...`` for documentation changes
-  * ``drivers: foo:`` for ``foo`` driver changes
-  * ``Bluetooth: Shell:`` for changes to the Bluetooth shell
-  * ``net: ethernet:`` for Ethernet-related networking changes
-  * ``dts:`` for treewide devicetree changes
-  * ``style:`` for code style changes
-
-  If you're not sure what to use, try running ``git log FILE``, where
-  ``FILE`` is a file you are changing, and using previous commits that
-  changed the same file as inspiration.
-
-[summary of change]
-  The ``[summary of change]`` part should be a quick description of
-  what you've done. Here are some examples:
-
-  * ``doc: update wiki references to new site``
-  * ``drivers: sensor: sensor_shell: fix channel name collision``
-
-Commit Message Body
-===================
-
-.. warning::
-
-   An empty commit message body is not permitted. Even for trivial
-   changes, please include a descriptive commit message body. Your
-   pull request will fail CI checks if you do not.
+.. _contribution_tips:
 
-This part of the commit should explain what your change does, and why
-it's needed. Be specific. A body that says ``"Fixes stuff"`` will be
-rejected. Be sure to include the following as relevant:
+Contribution Tips
+=================
 
-* **what** the change does,
-* **why** you chose that approach,
-* **what** assumptions were made, and
-* **how** you know it works -- for example, which tests you ran.
+The following is a list of tips to improve and accelerate the review process of
+Pull Requests. If you follow them, chances are your pull request will get the
+attention needed and it will be ready for merge sooner than later:
 
-Each line in your commit message should usually be 75 characters or
-less. Use newlines to wrap longer lines. Exceptions include lines
-with long URLs, email addresses, etc.
-
-For examples of accepted commit messages, you can refer to the Zephyr GitHub
-`changelog <https://github.com/zephyrproject-rtos/zephyr/commits/main>`__.
-
-If the change addresses a GitHub issue, include a line of the form:
-
-.. code-block:: none
-
-   Fixes #[issue number]
-
-Where ``[issue number]`` is the relevant GitHub issue's number. For
-example:
-
-.. code-block:: none
-
-   Fixes: #1234
-
-Signed-off-by: ...
-==================
-
-.. tip::
-
-   You should have set your :ref:`git-name-and-email`
-   already. Create your commit with ``git commit -s`` to add the
-   Signed-off-by: line automatically using this information.
+#. Make sure you are on top of latest main branch::
 
-For open source licensing reasons, your commit must include a
-Signed-off-by: line that looks like this:
+     git fetch upstream
+     git rebase upstream/main
 
-.. code-block:: none
+#. Run compliance tests locally and fix any issues before submitting a PR
```

Comment:
this is somewhat redundant since the CI runs this anyway, already covered by the "Watch early CI results immediately after submissions and fix issues as they are discovered"

Reply:
Item removed.